test/test_torch.py

```diff
@@ -15227,17 +15227,36 @@ def test_complex_type_conversions(self, device):
                 else:
                     self.assertEqual(from_tensor, to_tensor, exact_dtype=False)
 
-    @onlyCPU
     @dtypes(torch.complex64, torch.complex128)
     def test_complex_unsupported(self, device, dtype):
-        inp = torch.tensor((1 + 1j), device=device, dtype=dtype)
-        # Note: this is consistent with NumPy
+        t = torch.tensor((1 + 1j), device=device, dtype=dtype)
+        # Note: these are expected failures consistent with NumPy
+        with self.assertRaises(RuntimeError):
+            torch.floor(t)
+        with self.assertRaises(RuntimeError):
+            torch.ceil(t)
+        with self.assertRaises(RuntimeError):
+            torch.trunc(t)
+
+        # Tests clamp variants with complex inputs
```

Comment:
I don't think this is our current understanding.  Perhaps link to the discussion issue?

Reply:
Yep! This comment needs to be updated.